src/openmrs-backend-dependencies.ts

```diff
@@ -0,0 +1,4 @@
+export const backendDependencies = {
+  "webservices.rest": "2.24.0",
+  fhir: "1.18.0"
```

Comment:
Does esm-home actually use fhir? All I see is [openmrsFetch](https://github.com/openmrs/openmrs-esm-home/blob/0d41593c45315fa24e47aa3e7d1f48e1be62a43a/src/patient-search/patient-search.resource.tsx#L4), which is not using fhir.

Reply:
removed fhir, merging this now.